src/exo3 - Sort with Ord/exo3.ts

```diff
@@ -34,13 +41,19 @@ import { unimplemented } from '../utils';
 // expose some pre constructed instances of `Ord<T>` for said primitives such as
 // `string.Ord: Ord<string>` or `number.Ord: Ord<number>`.
 
-export const sortStrings: (
+export const sortStrings = (
```

Comment:
Here, using `flow` instead of `pipe` would simplify a lot the code, and you could actually even use neither `flow` or `pipe` in this example :) let's call to do it together if you want!

Reply:
I think I fixed this one :)